server/app/routes/api/v1/annotations/index.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
```

Comment:
To me this seems like 2 different PRs -- 1 for the seeding and another for the routes being built out like I'm starting to see below